tsconfigBase.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
[Nit] I wonder if we should put this at `shared/tsconfig.json` so that we get the hints that come with a `tsconfig.json` file?

Reply:
yes done

Reply:
Also incorporated the new flags from pocketderm repo open PR